src/OpenSage.Game/Data/W3d/W3dChunkType.cs

```diff
@@ -136,6 +136,7 @@ public enum W3dChunkType
             W3D_CHUNK_AGGREGATE_INFO,                                       // references to 'contained' models
             W3D_CHUNK_TEXTURE_REPLACER_INFO,                                    // information about which meshes need textures replaced
             W3D_CHUNK_AGGREGATE_CLASS_INFO,                                 // information about the original class that created this aggregate
+            W3D_CHUNK_AGGREGATE_UNKNOWN = 393728,                           // Unknown chunk found in enb asset ("npc14b.w3d")
```

Comment:
This seems more like a corrupt chunk. Is the model that uses this ever referenced by the game itself?

Reply:
Seems like this is indeed corrupt chunks. There are only 2 enb assets that contain this corruption and I can find no reference to them being used in the game or the emulator.